client/src/components/ReviewTile.js

```diff
@@ -1,10 +1,16 @@
-import React, { useState, useEffect } from "react";
+import React, { useState } from "react";
+import DeleteButton from "./DeleteReview";
 
-const ReviewTile = ({ review, castVote, currentUser }) => {
-  const { id, author, subject, body, rating, votes } = review
+const ReviewTile = ({ review, castVote, currentUser, reviews, restaurant, setRestaurant }) => {
+  const { id, author, subject, body, rating, votes, userId } = review
   const { score, currentUserVote } = votes
   const [voteState, setVoteState] = useState(currentUserVote.value)
 
+  let deleteButton = ""
```

Comment:
leave variable as undefined, does not need to be an empty string because it will become a react component of HTML
```js
let deleteButton;
```